core/aggregation.go

```diff
@@ -151,7 +157,23 @@ func (a *StdSignatureAggregator) ReceiveSignatures(
 
 	for numReply := 0; numReply < numOperators; numReply++ {
 		var err error
-		r := <-messageChan
+
+		var r SigningMessage
+
+		var contextExpired bool
+		select {
+		case r = <-messageChan:
+		case <-attestationCtx.Done():
+			remainingReplies := numOperators - numReply
+			a.Logger.Errorf(
```

Comment:
Should it be an error or would warning make more sense?

Reply:
changed to a warning